src/Settings/Video.SettingsGroup.json

```diff
@@ -128,6 +128,14 @@
     "longDesc":  "If this option is enabled, the rtpjitterbuffer is removed and the video sink is set to assynchronous mode, reducing the latency by about 200 ms.",
     "type":             "bool",
     "default":     false
+},
+{
+    "name":             "forceVideoDecoder",
+    "shortDesc":        "Force specific category of video decode",
+    "longDesc":         "Force the change of prioritization between video decode methods, allowing the user to force some video hardware decode plugins if necessary.",
+    "type":             "string",
```

Comment:
I was unable to find something that matches your description in QGC and in the *.json files of the settings, there is no "unsigned" values that I could find, only "win", "float", "int32", "SimpleItem".
Can you point a settings that matches what are you talking about  ?

Reply:
* Supported types are here: https://github.com/mavlink/qgroundcontrol/blob/master/src/FactSystem/FactMetaData.cc#L96
* Example usage: https://github.com/mavlink/qgroundcontrol/blob/master/src/Settings/Video.SettingsGroup.json#L63

Reply:
Thanks @DonLakeFlyer, but how is it possible to control the visibility of each option based in the operating system ?
From what I can see, the fact system does not have a function to remove items only to add, I also was unable to find any configuration regard the operating system.

Reply:
I was only commenting to the using of a string versus an enum for a settings. The way you were setting the visible property as needed in the settings C++ code was correct. The FactSystem json parser does not have the ability to set visible on an OS by OS base directly in the json. The only support sort of like that is `default` versus `mobileDefault` (https://github.com/mavlink/qgroundcontrol/blob/master/src/FactSystem/FactMetaData.cc#L136).

Reply:
Hi @DonLakeFlyer, thanks, this should be done in the last version, I'm defining the default configuration in the settings file and the others in the VideoSettings class, where the enumInfo is added by the operating system, this will save the configuration as numbers as requested and also control each possible value based in the operating system.

Reply:
Sorry I wasn't reading your question correctly. I thought you were still talking about `visible`. The way to handle an enum list which is different based on OS is to include the full list in the json. And then on the C++ side in the settings code you ask for the enum info, remove the ones which are inappropriate for the OS and then stuff it back in. This way the translation mechanism uses the standard json translation mechanism. It also removes confusion around some part of the enum in one place and some part in another place. I believe there are examples of this pattern in other settings someplace.

Reply:
Hi @DonLakeFlyer, no problem, since I was unable to find how to remove such items, I have created a function in the fact system to do such thing, I hope that is ok.